.github/workflows/health.yaml

```diff
@@ -0,0 +1,13 @@
+name: Health
+on:
+  pull_request:
+    branches: [ main, master ]
+    types: [opened, synchronize, reopened, labeled, unlabeled]
+jobs:
+  health:
+    uses: dart-lang/ecosystem/.github/workflows/health.yaml@main
+    with:
+      use-flutter: true
+      checks: "version,changelog,license,breaking,do-not-submit,leaking"
```

Comment:
can we limit the changelog check to certain packages? we don't maintain a changelog for packages/devtools_app or packages/devtools_test since these are not published

Reply:
Yes you can, there is an `ignore_packages` option

Reply:
I think this ignores _all_ of the checks though. I am also not sure if it is just smart enough to not check packages which have `publish_to: none`, cc @devoncarew ?

Reply:
cc @mosuem for the health check / changelog w/ `publish_to: none` question

Reply:
After checking: There is already [a filter](https://github.com/dart-lang/ecosystem/blob/49ec7130eb8bdae77eb934ac6b33caf5321cf498/pkgs/firehose/lib/src/health/changelog.dart#L55) in place for packages without a changelog.

Reply:
It also looks like version checks are skipped for `publish_to: none` packages